src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --font-texto-padrao: 'Roboto', sans-serif;
+  --cor-texto-padrao: #000;
+  --cor-ative: #00acdc;
```

Comment:
Change to english to keep the pattern on all code.

You can change to `--default-text-font`, `--default-text-color` and `--active-color` for example.